packages/load-script/src/callback-handler.ts

```diff
@@ -75,16 +78,14 @@ export function executeCallbacks( url, error = null ) {
 	}
 }
 
-export function handleRequestSuccess() {
-	const url = this.getAttribute( 'src' );
+export function handleRequestSuccess( this: HTMLScriptElement ) {
+	const url = this.src;
 	debug( `Handling successful request for "${ url }"` );
 	executeCallbacks( url );
-	this.onload = null;
 }
 
-export function handleRequestError() {
-	const url = this.getAttribute( 'src' );
+export function handleRequestError( this: HTMLScriptElement ) {
+	const url = this.src;
 	debug( `Handling failed request for "${ url }"` );
 	executeCallbacks( url, new Error( `Failed to load script "${ url }"` ) );
-	this.onerror = null;
```

Comment:
This should be redundant, handlers added with `addEventListener` and `{ once: true }`.

I'd only expect a load/error handler to be called once anyways 🤔